package.json

```diff
@@ -69,9 +68,9 @@
     "package-json-reducer": "^1.0.18",
     "prettier": "^2.3.0",
     "rimraf": "^3.0.2",
-    "semantic-release": "18.0.1",
-    "typedoc": "^0.17.0",
-    "typescript": "~3.8.3"
+    "semantic-release": "19.0.2",
+    "typedoc": "^0.22.13",
+    "typescript": "~4.5.5"
```

Comment:
Before we merge this, we might need to check this branch of the core with an actual SDK. There have been compilation problems with our generated code and `typescript` versions past v3.8.3